Upstream community updates for the policy collection (#392)
open-cluster-management-io/policy-collection: community/README.md

[Integrity Shield](./CM-Configuration-Management/policy-integrity-shield.yaml)| Use the Integrity Shield to protect the integrity of Kubernetes resources in a cluster (e.g. OpenShift). | See the [Integrity Shield documentation.](https://github.com/open-cluster-management-io/integrity-shield/blob/master/docs/ACM/README_ENABLE_ISHIELD_PROTECTION_ACM_ENV.md)
[Integrity Shield Events](./CM-Configuration-Management/policy-integrity-shield-events.yaml)| Use the Integrity Shield Events policy to show a status, which represents whether Integrity Shield has denied some requests in a cluster or not. | See the [Integrity Shield documentation.](https://github.com/open-cluster-management-io/integrity-shield/blob/master/docs/ACM/README_SHOW_INTEGRITY_STATUS_ON_ACM_UI.md)

> [Integrity Shield Observer](./CM-Configuration-Management/policy-integrity-shield-observer.yaml)| Integrity Shield Observer continuously verifies Kubernetes resource on cluster according ManifestIntegrityConstraint resources and exports the results to resources called ManifestIntegrityState. This policy is used to alert on any resource signature violations that have been identified by the observer. | This policy does not install Integrity Shield. This policy does require Integrity Shield to already be installed. See the [Integrity Shield documentation.](https://github.com/open-cluster-management-io/integrity-shield/blob/master/docs/ACM/README_SHOW_INTEGRITY_STATUS_ON_ACM_UI.md)

Reviewer: I think these need to stay as stolostron links

[OpenShift Kernel Configuration](./CM-Configuration-Management/policy-kernel-devel.yaml) | Use this policy to create OpenShift 4 machine configurations that install kernel development packages. The kernel development packages are needed for the Sysdig and Falco agents to integrate into the host kernel. | This policy is only valid for OpenShift 4.x.
[Volsync Persistent Data Replication](./CM-Configuration-Management/policy-persistent-data-management.yaml) | Use this policy to deploy the Volsync controller. Once the controller has been deployed, replication source and replication destination objects can be created allowing for persistent volume claims to be replicated. | See the [VolSync Replication](https://volsync.readthedocs.io) documentation for more information.

> [Setup-Subscription-Admin](./CM-Configuration-Management/policy-configure-subscription-admin-hub.yaml) | Use this policy to activate the Subscription-Admin feature in Open Cluster Management | See this Solution for more information about this feature [https://access.redhat.com/solutions/6010251](https://access.redhat.com/solutions/6010251)

Reviewer: This seems like it might be ACM specific

[policy-pod-placement](./CM-Configuration-Management/policy-pod-placement.yaml) | Ensures that a pod exists as specified. This policy uses the `Placement` kind rather than a `PlacementRule` to select managed clusters to deploy to. |
[Policy checking for ConfigurationPolicies stuck in a Terminating state](./CM-Configuration-Management/terminating-configpolicies.yaml) | Notifies that a ConfigurationPolicy was not able to delete some objects from its `PruneObjectBehavior` setting. Those related objects, and/or the ConfigurationPolicy might need to be manually cleaned up. |
[Policy to add node to an existing OpenShift cluster](./CM-Configuration-Management/policy-ztp-node-add.yaml) | Adds a new node to an existing ZTP cluster in Open Cluster Management.  Useful to scale capacity to existing clusters as a day 2 operation.  This policy will apply to the hub cluster and deploy a new `BareMetalHost` resource in the managed cluster's namespace. | The multicluster-engine operator must be set up and configured on the hub cluster.  This policy assumes that DHCP is available for the node to obtain network information.  BMH secrets must be created separately.

> [Policy to add node wtih static IP configurations to an existing OpenShift cluster](./CM-Configuration-Management/policy-ztp-node-add-static.yaml) | Adds a new node with a Static IP to an existing ZTP cluster in Open Cluster Management.  Useful to scale capacity to existing clusters as a day 2 operation.  This policy will apply to the hub cluster and deploy new `BareMetalHost` and `NMStateConfig` resources in the managed cluster's namespace. | The multicluster-engine operator must be set up and configured on the hub cluster. BMH secrets must be created separately. 

Reviewer: Another ZTP spot that might remain as ACM?

[Configure SriovNetwork ](./CM-Configuration-Management/policy-sriovnetwork-templatized.yaml) | Using hub templates configures the SriovNetwork resource with values customized to the target cluster , template-type: hub, template-functions : fromConfigMap, .ManagedClusterName | [Sriov Network Operator](https://github.com/openshift/sriov-network-operator) , [Go Templates functions](https://pkg.go.dev/text/template)

## Deploying community policies to your cluster

> While the policies in the [stable](../stable) folder are supported by Open Cluster Management Policy SIG, community policies are maintained by the open source community. You might need to deploy extra policy consumers in order for community policies to work as intended. If you are seeing the error `no matches for kind "<resource name>" in version "<group>/<version>"`, you must deploy the CustomResourceDefinition (CRD) for the policy before you create it. If some of the policies in this folder are not behaving properly, you must deploy the corresponding policy consumers to handle them.

Reviewer 2: ```suggestion
While the policies in the [stable](../stable) folder are supported by the Open Cluster Management Policy SIG, community policies are maintained by the open source community. You might need to deploy extra policy consumers in order for community policies to work as intended. If you are seeing the error `no matches for kind "<resource name>" in version "<group>/<version>"`, you must deploy the CustomResourceDefinition (CRD) for the policy before you create it. If some of the policies in this folder are not behaving properly, you must deploy the corresponding policy consumers to handle them.
```

> While the policies in the [stable](../stable) folder all have out-of-the-box support installed with Open Cluster Management, community policies are maintained by the open source community. You might need to deploy extra policy consumers in order for community policies to work as intended. If you are seeing the error `no matches for kind "<resource name>" in version "<group>/<version>"`, you must deploy the CustomResourceDefinition (CRD) for the policy before you create it. If some of the policies in this folder are not behaving properly, you must deploy the corresponding policy consumers to handle them.

Reviewer 2: Also, "deploy extra policy consumers" is not clear to me.
```suggestion
While the policies in the [stable](../stable) folder are supported Open Cluster Management Policy SIG, community policies are maintained by the open source community. You might need to deploy extra policy consumers in order for community policies to work as intended. If you are seeing the error `no matches for kind "<resource name>" in version "<group>/<version>"`, you must deploy the CustomResourceDefinition (CRD) for the policy before you create it. If some of the policies in this folder are not behaving properly, you must deploy the corresponding policy consumers to handle them.
```

### Policy consumers on operator hub

> Some policy consumers are packaged as [operators](https://coreos.com/operators/) and are available on the [Operator hub](https://operatorhub.io/). These consumers can simply be deployed by creating a policy with child [configuration policies](https://github.com/open-cluster-management-io/config-policy-controller) to handle the installation. The configuration policies might include the following information:

Reviewer 2: It might be worth noting that these often rely on Red Hat Operator Catalog.
